sdcontent/INSTRUCTIONS-README.txt

```diff
@@ -14,21 +14,21 @@ NOOBS INSTALLATION INSTRUCTIONS
       iv. Check that the SD card you inserted matches the one selected by the Tool
       v. Click the "Format" button
    c. Linux
-      i. We recommend using gparted (or the command line version parted)
-      ii. Format the entire disk as FAT32
+      i. We recommend using gparted (or the command line version, parted)
+      ii. Format the entire card as FAT32
    d. Cards over 32GB
-      i. Follow the instructions on https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md
+      i. Follow the instructions at https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md
 3. Extract the files contained in this NOOBS zip file.
-4. Copy the extracted files onto the SD card that you just formatted so that this file is at the root directory of the SD card. Please note that in some cases it may extract the files into a folder, if this is the case then please copy across the files from inside the folder rather than the folder itself.
-5. Insert the SD card into your Pi and connect the power supply.
+4. Copy the extracted files onto the SD card that you just formatted. The files must be in the root directory of the SD card, not in a subdirectory within this. If the files are in a subdirectory on the SD card, then you will not be able to start a Raspberry Pi from the SD card.
+5. Insert the SD card into the Raspberry Pi and connect the power supply.
 
-**Raspberry Pi 4 Model B:** display must be connected to HDMI port closest to power jack.
+**Note:** on the Raspberry Pi 4B you must connect the display to the micro HDMI port closest to the power jack, labelled HDMI 0.
 
 Your Pi will now boot into NOOBS and should display a list of operating systems that you can choose to install.
 If your display remains blank, you should select the correct output mode for your display by pressing one of the following number keys on your keyboard:
 1. HDMI mode - this is the default display mode.
 2. HDMI safe mode - select this mode if you are using the HDMI connector and cannot see anything on screen when the Pi has booted.
-3. Composite PAL mode - select either this mode or composite NTSC mode if you are using the composite RCA video connector.
+3. Composite PAL mode - select either this mode or composite NTSC mode if you are using a screen connected to the composite video port. (Either RCA or 'headphone' connector, depending on model of Raspberry Pi).

```

Comment:
Do you think it's worth mentioning here about enable_tvout being required to use composite video on the Pi4? WDYT @procount ?

Reply:
Good point. I'm not sure how NOOBS handles that. Does the parameter get passed down to Raspbian when it boots? It would be easier for inexperienced users of they could just add a file to the NOOBS partition to enable composite, like for SSH. Although NOOBS is effectively EOL now, as I understand it.

Reply:
> I'm not sure how NOOBS handles that. Does the parameter get passed down to Raspbian when it boots?

I dunno, I can't even remember the last time I used the composite output on any of my Pis, and I've never used composite output on my Pi 4B :confused: 
I pinged @procount because I think this is an area he might have more familiarity with?

> Although NOOBS is effectively EOL now, as I understand it.

Yeah, the way I understand it, the new Raspberry Pi Imager tool is now recommended over NOOBS.

Reply:
Viva PINN! 
Even I recommend using RPi Imager to flash PINN (See https://www.raspberrypi.org/forums/viewtopic.php?f=63&t=142574&start=975#p1638508) as it avoids so many issues with FAT32.
If the aim is just to install ONE OS, the RPIimager can be recommended. But it doesn't replace NOOBS' or PINN's abiity to multi-boot. Horses for courses!

(However, I have had a couple of instances where RPI imager will fail to verify, but Etcher verifies the exact same image 🤷‍♂️)

Re: RCA output - the behaviour has changed with Pi4, so I think it should be mentioned to avoid people wondering why RCA doesn't work without setting enable_tvout. 

Likewise the fact that NOOBS/PINN only works on HDMI-0 (not HDMI-1). It's a common issue on the forums.

Reply:
Thanks for the clarification.

By 'RCA' you of course mean 'TRRS' 😉

Reply:
Yeah, yeah - sorry - just reusing the above terminology without thinking :)

Reply:
> (However, I have had a couple of instances where RPI imager will fail to verify, but Etcher verifies the exact same image man_shrugging)

If that's reproducible with the latest v1.2 of RPi Imager could you please create an issue at https://github.com/raspberrypi/rpi-imager ?

Reply:
> Likewise the fact that NOOBS/PINN only works on HDMI-0 (not HDMI-1). It's a common issue on the forums.

Do you want to file a PR against https://www.raspberrypi.org/documentation/installation/noobs.md ? :slightly_smiling_face: 

As a side-note, I guess there's a bit of a balancing act between "how much" gets documented at each of
 * https://github.com/raspberrypi/noobs/blob/master/README.md
 * https://github.com/raspberrypi/noobs/blob/master/sdcontent/INSTRUCTIONS-README.txt
 * https://www.raspberrypi.org/documentation/installation/noobs.md
 * https://www.raspberrypi.org/help/noobs-setup/2/

Reply:
I was taking the view that whatever is in the readme should stay, since the user may not have an internet connection when they do the install

Reply:
> whatever is in the readme should stay

Absolutely! I wasn't suggesting removing any documentation, just acknowledging that it'll be a bit of a juggling act deciding how much to duplicate between the various locations that I was pointing at.

Reply:
> Do you want to file a PR against https://www.raspberrypi.org/documentation/installation/noobs.md ? 🙂

That page appears to be an older version of https://www.raspberrypi.org/help/noobs-setup/2/.

Also, looking at https://www.raspberrypi.org/help/, it has some very useful content, but it is spread across 3 separate silos:

- raspberrypi.org/help
- raspberrypi.org/learning
- projects.raspberrypi.org

and has a copy of the content from raspberrypi.org/documentation/ at the bottom of it.

So including the documentation section of the website, that's 4 separate information silos. That seems suboptimal to me. Also, each silo has different navigation and style. In particular, the menu across the top of the page is different on each silo.

There appears to be a very small number of pages in the 'help' section of the website. Perhaps it should be merged with the 'documentation' section, using the style of the 'help' section, since that style is more in keeping with the rest of the website, now the blog has been redesigned.

Reply:
> that's 4 separate information silos. That seems suboptimal to me.

Agreed. But AFAIK raspberrypi.org/documentation/ is the only part updated by Raspberry Pi Trading (via the https://github.com/raspberrypi/documentation repo), everything else is managed by Raspberry Pi Foundation.

Reply:
> > (However, I have had a couple of instances where RPI imager will fail to verify, but Etcher verifies the exact same image man_shrugging)
> 
> If that's reproducible with the latest v1.2 of RPi Imager could you please create an issue at https://github.com/raspberrypi/rpi-imager ?

@lurch - I updated to v1.2 and it seems to be ok now, so either a buggy old version or corrupt installation.  But that's based on only one sample point so far.

Reply:
Yeah, IIRC there was a bug in versions of RPi Imager before 1.2 where the verification would (incorrectly) fail if the image-size wasn't an exact-multiple of some blocksize.